partner_middlename/tests/test_name.py

```diff
@@ -0,0 +1,176 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# Copyright 2015 Grupo ESOC Ingeniería de Servicios, S.L.U.
+# Copyright 2015 Antiun Ingenieria S.L. - Antonio Espinosa
+
+from odoo.tests.common import TransactionCase
+
+from odoo.addons.partner_firstname.tests.base import MailInstalled
+
+
+class CompanyCase(TransactionCase):
```

Comment:
I believe this is the same test as in "partner_firstname", is redundant and can be removed.